libtransmission/announcer.cc

```diff
@@ -1490,7 +1477,7 @@ static void on_scrape_done(tr_scrape_response const* response, void* vsession)
                 {
                     char* scheme = nullptr;
                     char* host = nullptr;
-                    int port;
+                    auto port = int{};
```

Comment:
now that just looks weird. Maybe

```
int port{};
```

is better?
```

Reply:
This is (trying to) use [almost always auto](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/), which does look weird if you're not used to it.